crates/store/re_video/src/decode/mod.rs

```diff
@@ -114,6 +117,27 @@ pub trait SyncDecoder {
     fn reset(&mut self) {}
 }
 
+#[cfg(not(target_arch = "wasm32"))]
+pub fn new_decoder(
```

Comment:
Taken from https://github.com/rerun-io/rerun/pull/7658 - this is a step in the direction of supporting more decoders.